### PR TITLE
upgraded to latest can-feathers-connect that fixes session auth bug (…

### PR DESCRIPTION
**Pull request by ilyavf**
…a new property session.isAuthenticated)

The bug showed up when refreshing browser and getting nonAuthenticated error.

### CODE REVIEW

File: public/app.js [hunk 1]
```diff
@@ -42,7 +42,8 @@ const AppViewModel = DefineMap.extend({
   page: {
     serialize: true,
     get (page) {
-      if (this.session) {
+      var session = this.session;
+      if (session && session.isAuthenticated) {
```
Comment (justinbmeyer) [inline public/app.js]: can you add an inline comment explaining why this change is needed?

Comment (ilyavf) [inline public/app.js]: fixed
